backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -28,10 +28,34 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
 
+[id="issues-with-velero-and-admission-workbooks"]
+== Issues with Velero and admission webhooks
+
+Velero has limited abilities to resolve admission webhook issues during a restore. If you have workloads with admission webhooks, you might need to use an additional Velero plug-in or make changes to how you restore the workload.
+
+Typically, workloads with admission webhooks require you to create a resource of a specific kind first. This is especially true if your workload has child resources because admission webhooks typically block child resources.
+
+For example, creating or restoring a top-level object such as `service.serving.knative.dev` typically creates child resources automatically. If you do this first, you will not need to use Velero to create and restore these resources. This avoids the problem of child resources being blocked by an admission webhook that Velero might use.  
+
+[id="velero-restore-workarounds-for-workloads-with-admission-webhooks"]
+=== Restoring workarounds for Velero backups that use admission webhooks
+
+This section describes the additional steps required to restore resources for several types of Velero backups that use admission webhooks.
+
+include::modules/migration-debugging-velero-admission-webhooks-knative.adoc[leveloffset=+3]
+include::modules/migration-debugging-velero-admission-webhooks-ibm-appconnect.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../architecture/admission-plug-ins.adoc[Admission plug-ins]
```

Comment:
This xref is missing an anchor

Reply:
I want to point to the start of "Admissions plug-ins," not "About admission plug-ins"  -- that is, I want to reference the assembly "admission-plug-ins.adoc," not a module that is included in it. Do assemblies have anchors (or contexts)?

Reply:
Hi @RichardHoch - yes assemblies still do have anchors, and all xrefs should have an anchor (even ones to assemblies). You'll have to check the actual assembly file to find the `id` at the top to use.

But I'm totally fine with merging this and following up on this in a separate PR.